config/monitoring/stopPrometheus.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+docker rm  prometheus
```

Comment:
When used gives:

`Error response from daemon: You cannot remove a running container <containerId>. Stop the container before attempting removal or force remove`

where `<containerId>` is the Id of the running container.

should have before:

`docker stop prometheus`